Ordering/Contracts/OrderManagement/OrderItem.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Contracts
+{
+    public class OrderItem : IOrderItem
+    {
+        public static int CurrentItemID = 0;
+
+        public int ItemID { get; }
+        public string Name { get; set; }
+        public string ItemDescription { get; set; }
+        public double Price { get; set; }
+        public DateTime DateLastOrdered { get; set; }
```

Comment:
Why the date last ordered and not order date?

Reply:
I have decided not to include this variable